src/main/java/hr/inovatrend/petManager/Controller/UserController.java

```diff
@@ -54,4 +54,15 @@ private String userEdit(@PathVariable("id") Long id, @ModelAttribute User user)
     }
 
 
+    @GetMapping("/delete/{id}")
```

Comment:
Koristiti @DeleteMapping ( drugi način @RequestMapping(method = RequestMethod.DELETE) )

Reply:
a kako ce deletemapping preko thymeleaf-a?
zar nisu sve <a> reference get by default?

Reply:
Ok, za sada nek' ostane tako, za ubuduće znaš da se treba koristiti delete mapping za delete (kod RESTa).